# Exception in thread “main“ redis.clients.jedis.exceptions.JedisConnectionException Failed to create
[Exception in thread “main“ redis.clients.jedis.exceptions.JedisConnectionException Failed to create](https://aiwithcloud.com/2022/09/19/exception_in_thread_main_redis-clients-jedis-exceptions-jedisconnectionexception_failed_to_create/)